### ISSUE TITLE
xml.format.maxLineWidth is ignored

### ISSUE THREAD

**Issue by phazei** [redhat-developer/vscode-xml#962]
I'm using VS Code 1.85.2 and I just installed the XML plugin.  When I format, it's wrapping all my text lines to a max of 121 columns.  I went to the settings and saw it was set at the default of 100, so I changed it to "0" to disable, and it still wraps everything to 121 columns.  I'm not sure where it's getting that from.

**Comment by angelozerr** [comment]
That's strange, could you share your XML please.

**Comment by phazei** [comment]
I thinks this is it:
{
    "gitlens.hovers.currentLine.over": "line",
    "gitlens.views.repositories.location": "gitlens",
    "gitlens.views.fileHistory.location": "gitlens",
    "gitlens.views.lineHistory.location": "gitlens",
    "gitlens.views.compare.location": "gitlens",
    "gitlens.views.search.location": "gitlens",
    "git.allowForcePush": true,
    "diffEditor.renderSideBySide": false,
    "workbench.startupEditor": "newUntitledFile",
    "liveshare.allowGuestDebugControl": true,
    "workbench.colorTheme": "Visual Studio Dark",
    "redhat.telemetry.enabled": false,
    "editor.renderWhitespace": "none",
    "xml.format.maxLineWidth": 0,
    "xml.fileAssociations": [

    ],
    "auto-build.defaultEnv.name": "LPC1769",
    "xml.colors": [
        
    ]
}

**Comment by angelozerr** [comment]
I mean your XML file please.

**Comment by phazei** [comment]
Ah, yeah, that makes sense too 😅
[0000_Information.xhtml.txt](https://github.com/redhat-developer/vscode-xml/files/14074516/0000_Information.xhtml.txt)

I had to add "txt" to the extension for it to let me attach it.  It's a file from a epub book.  In the process of getting this file I realized something.  If I open the file in VS Code and `alt-shift-f` it exhibits the issue with the `xhtml` extension, but if I just paste it into a new doc, or rename it to `xml`, then it does seem to work.  The thing is, there's hundreds of chapters and I wouldn't want to have to rename them all.
